
Traceback (most recent call last):
  File "evaluate_agents.py", line 194, in <module>
    evaluate_perf(agent, env, seeds, n_episodes_to_evaluate, save_dir)  
  File "evaluate_agents.py", line 65, in evaluate_perf
    mean_acc_rewards = evaluate_agent(agent, env, n_episodes_to_evaluate)
  File "evaluate_agents.py", line 29, in evaluate_agent
    next_obs, reward, done, _ = env.step(action)
  File "XXX/COMP579_project/environments.py", line 42, in step
    next_obs, reward, done, info = self.env.step(action)
  File "XXX/.conda/envs/COMP579_project/lib/python3.7/site-packages/gym/wrappers/time_limit.py", line 17, in step
    observation, reward, done, info = self.env.step(action)
  File "XXX/.conda/envs/COMP579_project/lib/python3.7/site-packages/gym/wrappers/order_enforcing.py", line 13, in step
    observation, reward, done, info = self.env.step(action)
  File "XXX/.conda/envs/COMP579_project/lib/python3.7/site-packages/gym/envs/mujoco/hopper.py", line 13, in step
    self.do_simulation(a, self.frame_skip)
  File "XXX/.conda/envs/COMP579_project/lib/python3.7/site-packages/gym/envs/mujoco/mujoco_env.py", line 136, in do_simulation
    raise ValueError("Action dimension mismatch")
ValueError: Action dimension mismatch
